tests/end_to_end/manual.py

```diff
@@ -81,19 +81,32 @@
         device.close()
         exit(8)
 
+    image_type = (
+        dai.ImgFrame.Type.BGR888i
+        if device.getPlatform().name == "RVC4"
+        else dai.ImgFrame.Type.BGR888p
+    )
```

Comment:
Can we set the type based on ˙dai_type˙ field in the NNArchive if present otherwise use this as default?

Reply:
Will be addressed in the future because dai has not yet implemented it.